README.md

```diff
@@ -78,7 +78,7 @@ impl Buildpack for HelloWorldBuildpack {
     }
 
     // Similar to detect, this method will be called when the CNB lifecycle executes the 
-    // build phase.
+    // `bin/build` phase.
```

Comment:
As commented on #199, "build phase" is a proper CNB term and the docs match what happens during that phase.